qiskit_ibm_runtime/options.py

```diff
@@ -104,6 +104,12 @@ class Options:
     """Options for the primitive programs.
 
     Args:
+        max_execution_time: Maximum execution time in seconds. This overrides
+            the max_execution_time of the program and cannot exceed it.
```

Comment:
I find this description a bit confusing. What does "overrides the max_execution_time of the program" mean? Does it update the `max_execution_time` metadata of a program? I don't think that's how this works?